<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dashboard</class>
 <widget class="QDialog" name="dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PolliTech</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../VVM.qrc">
    <normaloff>:/backgrounds/Smol logo no bg.png</normaloff>:/backgrounds/Smol logo no bg.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #f8f9fa, stop:1 #e9ecef);
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox_userManagement">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>411</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	font: bold 16pt &quot;Segoe UI&quot;;
    border: 1px solid #0A1C3A;
    border-radius: 8px;
    margin-top: 30px;
    padding: 15px;

    background-color: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #f5f5f5,
        stop:1 #e0e0e0
    );
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 10px;
    color: #0A1C3A;
    border-radius: 4px;
}</string>
   </property>
   <property name="title">
    <string>USER MANAGEMENT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>81</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>81</width>
        <height>81</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../VVM.qrc">:/icons/buttons/icons/user logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="manage_voter_button">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Add or Remove Voters&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font: 600 9pt &quot;Segoe UI&quot;;
	background-color: #f8f8f8;
	color: black;
	font-size: 16px;
	border: 1px solid black;
	border-radius: 5px;
	padding: 5px;
}
QPushButton:hover {
background-color: #e6e6e6;
}</string>
        </property>
        <property name="text">
         <string>MANAGE VOTERS</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="manage_candidate_button">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Add or Remove Candidates&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font: 600 9pt &quot;Segoe UI&quot;;
	background-color: #f8f8f8;
	color: black;
	font-size: 16px;
	border: 1px solid black;
	border-radius: 5px;
	padding: 5px;
}
QPushButton:hover {
background-color: #e6e6e6;
}</string>
        </property>
        <property name="text">
         <string>MANAGE CANDIDATES</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="manage_admins_button">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Add or Remove Admins&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font: 600 9pt &quot;Segoe UI&quot;;
	background-color: #f8f8f8;
	color: black;
	font-size: 16px;
	border: 1px solid black;
	border-radius: 5px;
	padding: 5px;
}
QPushButton:hover {
background-color: #e6e6e6;
}</string>
        </property>
        <property name="text">
         <string>MANAGE OTHER ADMINS</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_resultsNcontrol">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>411</width>
     <height>163</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	font: bold 16pt &quot;Segoe UI&quot;;
    border: 1px solid #0A1C3A;
    border-radius: 8px;
    margin-top: 30px;
    padding: 15px;

    background-color: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #f5f5f5,
        stop:1 #e0e0e0
    );
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 10px;
    color: #0A1C3A;
    border-radius: 4px;
}</string>
   </property>
   <property name="title">
    <string>ELECTION CONTROL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>81</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>81</width>
        <height>81</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../VVM.qrc">:/icons/buttons/icons/dashboard.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="view_live_vote_button">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;View live vote count&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font: 600 9pt &quot;Segoe UI&quot;;
	background-color: #f8f8f8;
	color: black;
	font-size: 16px;
	border: 1px solid black;
	border-radius: 5px;
	padding: 5px;
}
QPushButton:hover {
background-color: #e6e6e6;
}</string>
        </property>
        <property name="text">
         <string>VIEW VOTE COUNT</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="start_election_button">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Control Election&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font: 600 9pt &quot;Segoe UI&quot;;
	background-color: #f8f8f8;
	color: black;
	font-size: 16px;
	border: 1px solid black;
	border-radius: 5px;
	padding: 5px;
}
QPushButton:hover {
background-color: #e6e6e6;
}</string>
        </property>
        <property name="text">
         <string>START/STOP ELECTION</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>430</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>111</width>
     <height>111</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>111</width>
     <height>111</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../VVM.qrc">:/backgrounds/256x256 logo w name no bg.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="LogoutButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>99</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font: 600 9pt &quot;Segoe UI&quot;;
    font-size: 12px;
    color: white;

    background-color: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 rgb(220, 60, 50), 
        stop:1 rgb(180, 30, 30)
    );

    border: 1px solid #5A0A0A; 
    border-radius: 14px;
    padding: 5px 15px;
}

QPushButton:hover {
    background-color: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 rgb(240, 80, 70),
        stop:1 rgb(200, 40, 40)
    );
}

QPushButton:pressed {
    background-color: rgb(160, 20, 20);
    padding-top: 6px;
    padding-bottom: 4px;
}

QPushButton:focus {
    outline: none;
    border: 2px solid #8B0000;
}
</string>
   </property>
   <property name="text">
    <string>LOGOUT</string>
   </property>
   <property name="icon">
    <iconset resource="../VVM.qrc">
     <normaloff>:/icons/buttons/icons/power_icon.png</normaloff>:/icons/buttons/icons/power_icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>353</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 700 9pt &quot;Segoe UI&quot;;
	font-size: 35px;
	color: #0A1C3A;
}</string>
   </property>
   <property name="text">
    <string>ADMIN DASHBOARD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>manage_voter_button</tabstop>
  <tabstop>manage_candidate_button</tabstop>
  <tabstop>view_live_vote_button</tabstop>
 </tabstops>
 <resources>
  <include location="../VVM.qrc"/>
 </resources>
 <connections/>
</ui>
